src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/fine/mapper/FineWfExecutionConfigMapper.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.pu.workflow.wf.debtposition.custom.fine.mapper;
+
+import it.gov.pagopa.payhub.activities.dto.debtposition.syncwfconfig.FineWfExecutionConfig;
+import it.gov.pagopa.payhub.activities.dto.debtposition.syncwfconfig.GenericWfExecutionConfig;
+
+public class FineWfExecutionConfigMapper {
+
+  private FineWfExecutionConfigMapper() {}
+
+  public static GenericWfExecutionConfig mapFineWfExecutionConfigToGenericWfExecutionConfigForIsNotified(FineWfExecutionConfig fineConfig) {
+    return GenericWfExecutionConfig.builder()
+      .ioMessages(GenericWfExecutionConfig.IONotificationBaseOpsMessages.builder()
+        .created(fineConfig.getIoMessages().getNotified())
```

Comment:
it could be called also on update use case
```suggestion
        .created(fineConfig.getIoMessages().getNotified())
        .updated(fineConfig.getIoMessages().getNotified())
```